tests/test_page.py

```diff
@@ -238,6 +238,13 @@ def test_extract_text_single_quote_op():
         page.extract_text()
 
 
+def test_no_ressources_on_text_extract():
+    url = "https://raw.githubusercontent.com/eagletrt/wiki/0f3f16309604f665a47595c890d15af1b3aec6d6/fenice-telemetry-tx/PCB%20Outputs/Pdf/Edge%20Mount%20SMA/TelemetryTX_EM.pdf"
```

Comment:
Better to not rely on a random third-party repo I think, and instead attach the PDF into the ticket and can link that:
```suggestion
    url = "https://github.com/py-pdf/PyPDF2/files/9428434/TelemetryTX_EM.pdf"
```

Reply:
use of the referenced file from the issue. thanks for the recommendation